packages/inter-protocol/src/price/priceOracleAdmin.js

```diff
@@ -36,9 +35,10 @@ const oracleAdminKind = makeKindHandle('OracleAdmin');
  * @param {HeldParams} heldParams
  * @returns {State}
  */
-const initState = heldParams => {
+const initState = ({ endingRound, roundPowers }) => {
   return {
-    ...heldParams,
+    endingRound,
+    roundPowers,
```

Comment:
Okay, now I'll ask why these are saved in `state`. They're not accessed in `pushPrice()` or `getStatus()`, and nothing else provides access to `state`.  Or did I miss something?

Reply:
Ah. `endingRound` was supposed to be included in `getStatus()`. I've added a `returns` declaration so that errors and then fixed it.

`roundPowers` I think you missed https://github.com/Agoric/agoric-sdk/blob/fa9f5232ee70ebce1fd2c2facf9a779182d11705/packages/inter-protocol/src/price/priceOracleAdmin.js#L68

Reply:
follow up on `endingRound`: removed in fcdb451d15293d6f99df47ccf0124f077789091a